src/pages/Video/Video.page.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import VideoPlayer from '../../components/VideoPlayer';
+import RelatedVideos from '../../components/RelatedVideos'
+import Navbar from '../../components/Navbar';
+import { useHistory } from 'react-router-dom';
+
+const VideoPage = () => {
+    const history = useHistory()
+    const updateSearch = (childData) =>{
+        history.push(`/q:${childData}`)
```

Comment:
The standard URL formatting is to use the value using paths or using query parameters, but you're having a combination of both. I recommend using `<path>?q=${childData}` or `<path>/q/{childData}`

Reply:
Great! Thanks for the tip.